core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -2410,6 +2410,12 @@ public String toString() {
      * Represents the state of the upgrade activity of Jenkins core.
      */
     public final class HudsonUpgradeJob extends DownloadJob {
+
+        @Deprecated
```

Comment:
the other classes that dextend Download JOb are also affected, probably best to add backwards compatability to them all?